DirectEmployers/MyJobs
# [PD-3027][PD-3163] Site traffic report.

## What's this PR do?
Adds the Site Traffic report

## Screenshots (if appropriate)
![site-traffic-report](https://cloud.githubusercontent.com/assets/2728248/25283996/b44275a6-2683-11e7-82ed-758c1259e1c9.png)

## Where should the reviewer start?
There is a migration to add the report. Migrations are pretty out of hand this sprint...
Most changes were on the analytics API/utils side.
Minimal changes were done to get the analytics_event datasource working.

## How should this be manually tested?
Run the migrations.
Run `python manage.py make_data`.
Navigate to the report intentions page - site traffic should be an option.
Click through to the site traffic report and mess around. Stuff should work.

## Any background context you want to provide?
CSS for the table is still messed up for this PR and the next one I'm going to put up. 
I'm going to bother @mwarnsleyDE about what he thinks we should do for that. So a fix for that is coming separately.
